lib/l10n/app_de.arb

```diff
@@ -1,6 +1,24 @@
 {
     "@@locale": "de",
 
+    "price": "price: {amount}€",
```

Comment:
do we really want it this way? It's obvious that this is the price because of the currency symbol

Reply:
ok i'll change it